std/internal/scopebuffer.d

```diff
@@ -178,7 +180,7 @@ struct ScopeBuffer(T, alias realloc = /*core.stdc.stdlib*/.realloc)
      * If $(D const(T)) can be converted to $(D T), then put will accept
      * $(D const(T)[]) as input. It will accept a $(D T[]) otherwise.
      */
-    private alias CT = Select!(is(const(T) : T), const(T), T);
+    package alias CT = Select!(is(const(T) : T), const(T), T);
```

Comment:
CT has the docs for the following `put`, which uses `ditto`. As CT is private, ddoc ignores its docs and the ditto makes `put(CT[])` ditto the first overload of `put`. (Maybe ddoc could be changed to be more helpful here).